fabric_plugin/tasks.py

```diff
@@ -239,7 +264,7 @@ def _hide_or_display_results(hide_value, result):
     elif hide_value == "out":
         _log_output(ctx, result.stderr, prefix='<err> ')
     elif hide_value == 'err':
-        _log_output(ctx, result.stdout, prefix='<err> ')
+        _log_output(ctx, result.stdout, prefix='<out> ')
```

Comment:
Why should we log `err` as `<out>`?

Reply:
based on the logic it was an old bug , see we want to hide error , and show out , see the `result.stdout`